Qt/mainwindow.h

```diff
@@ -122,28 +132,18 @@ private slots:
 	// Sound
 	void audioAct() { g_Config.bEnableSound = !g_Config.bEnableSound; }
 
+	// Cheats
+	void cheatsAct() { g_Config.bEnableCheats = !g_Config.bEnableCheats; }
+
 	void fullscrAct();
 	void raiseTopMost();
 	void statsAct() {
 		g_Config.bShowDebugStats = !g_Config.bShowDebugStats;
 		NativeMessageReceived("clear jit", "");
 	}
+	void VSyncAct() { g_Config.bVSync = !g_Config.bVSync; }
 	void showFPSAct() { g_Config.iShowFPSCounter = !g_Config.iShowFPSCounter; }
```

Comment:
Ideally, this should toggle between 0 and 3, not 0 and 1.  That's what it does on Windows - 3 means show FPS and Speed.

-[Unknown]

Reply:
In Windows version, menu doesn't correspond internal settings: it has 2 states and turns both Speed and FPS when active.
![1](https://user-images.githubusercontent.com/14265316/79079551-7d1d9800-7d18-11ea-8ef3-49a075a25ba4.png)
![2](https://user-images.githubusercontent.com/14265316/79079552-8575d300-7d18-11ea-9866-4f943b4f31db.png)
It would be better to fix then both in further PRs.

Reply:
Well, I think the idea is - it's not really that necessary to be able to select between all four options in the menu.  It (and other settings) are available in the cross-platform UI within the emulator if you need it.

For example, we definitely wouldn't want to make the Windows menu a maze with every single possible setting shown there.

That's why we don't show all the possible FPS options either.

-[Unknown]

Reply:
I do think that if we show a setting in the platform-specific UI, it's nice if the choices match those in the cross-platform one - this kind of inconsistency isn't super elegant. But there's indeed also some merit in keeping things simple.

Reply:
The idea of it is reasonable, but disrepance of these menus is a little confusing. And the top menu is already the maze (compared to RPCS3, it looks more like LibreOffice). It wouldn't hurt to slim it down, I guess, but it's the subject of another discussion.
OK, I'll change it to Windows-specific way.